src/uthash.h

```diff
@@ -105,6 +102,30 @@ typedef unsigned char uint8_t;
 #define uthash_expand_fyi(tbl)            /* can be defined to log expands   */
 #endif
 
+#ifndef HASH_NOMEM_OK
+#define HASH_NOMEM_OK 0
+#endif
+
+#if HASH_NOMEM_OK
+/* malloc failures can be recovered from */
+
+#ifndef uthash_fatal
+#define uthash_fatal(obj) do{}while(0)
```

Comment:
`where`: I think it should be a string literal; and beyond that, if we can provide more information, we probably should.
Having it be the same as what's passed to `HASH_FSCK` is certainly fine with me.